doc/start/quick-start-preflight.rst

```diff
@@ -122,7 +122,11 @@ On CentOS / RHEL, execute::
 
 On Debian / Ubuntu, execute::
 
-	sudo apt install ntp
+	sudo apt install ntp 
```

Comment:
@zdover23 thanks for addressing this issue. i am wondering if you could help to improve the Debian side as well. as i think we should encourage user to use [ntpsec](https://www.ntpsec.org) instead of the classic `ntp`. and both `ntpsec` and `chrony` are available in current debian stable (buster) and ubuntu 18.04 and up.

so probably you could instead change

```sh
sudo apt install ntp
```

to 

> ```sh
> sudo apt install ntpsec
> ```
> or
> ```
> sudo apt install chrony
> ```
> 

?

Reply:
tchaikov,

I will update this commit with your suggestion and force push this commit.

Stand by for this update.